### ISSUE TITLE
Canceling edit of comment says "Are you sure you want to clear this form". Why not "Exit comment editor"? Current one is mighty confusing

### ISSUE THREAD

**Issue by royachiron** [steemit/condenser#2819]
![image](https://user-images.githubusercontent.com/23020420/40570720-7d1d5c0e-6042-11e8-964d-320adc09e69b.png)

**Comment by royachiron** [comment]
@roadscape I thank you for acknowledging the issue, though since I'm not a code guy I didn't know that this bit of text is for the post and comment editor. So now when you cancel editing a post, you are asked about exiting the comment editor. Maybe (sorry I didn't check that they're identical when I submitted this :/ ) change it to "exit the text editor"? or change it to any other appropriate text that is better than my suggestion?

**Comment by roadscape** [comment]
@royachiron IMO 'comment vs post' is not a critical distinction in this case, and it's much less confusing than the last message.

**Comment by royachiron** [comment]
I agree, it's the tiny perfectionist in me that wrote that. Comment can be used to describe the text on the blockchain and it makes perfect sense. Thanks for taking the time to take care of this tiny issue. Would appreciate if you look in my other suggestion here and give your opinion :)
